modules/light-clients/06-solomachine/solomachine_test.go

```diff
@@ -46,6 +46,18 @@ func TestSoloMachineTestSuite(t *testing.T) {
 	suite.Run(t, new(SoloMachineTestSuite))
 }
 
+func (suite *SoloMachineTestSuite) TestConnectionHandshake() {
```

Comment:
These tests couldn't be split out because they actually will fail on main (main is currently broken for solo machines)